### PR TITLE
[Selenium] Fix unstable selenium tests, add info about know issue

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR:
- fix unstable **DirectUrlFactoryWithRootFolderTest** and **DirectUrlFactoryWithSpecificBranchTest** tests from ```factory``` package;
- add info about #13893 known issue to **AddWorkspaceToOrganizationTest** selenium test;
- remove **YamlFileEditingTest** test from ```CheSuite.xml``` test suite(this test checks Che6 workspace functionality).

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

File: selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml [hunk 1]
```diff
@@ -32,7 +32,6 @@
           <class name="org.eclipse.che.selenium.languageserver.TypeScriptEditingTest"/>
           <class name="org.eclipse.che.selenium.languageserver.ClangFileEditingTest"/>
           <class name="org.eclipse.che.selenium.languageserver.ApacheCamelFileEditingTest"/>
-          <class name="org.eclipse.che.selenium.languageserver.YamlFileEditingTest"/>
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml]: Please, don't forget to remove test class if it is being removed from test suite.

**Comment by rhopp** [comment]
@SkorikSergey `DirectUrlFactoryWithSpecificBranchTest` failed. Could you please investigate the reason? May be some bug...

File: selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml (hunk 1 above)
Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/resources/suites/CheSuite.xml]: Done

**Comment by SkorikSergey** [comment]
> @SkorikSergey `DirectUrlFactoryWithSpecificBranchTest` failed. Could you please investigate the reason? May be some bug...

The reason of this failing is changes from https://github.com/eclipse/che/pull/13883. I will start ```ci-test``` after it fixed.

**Review by dmytro-ndp — APPROVED** [review]

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by musienko-maxim** [comment]
check webhook trigger

**Comment by musienko-maxim** [comment]
ci-test

**Review by nickboldt — APPROVED** [review]
Tentative +1. Probably better for QE to have final say here. 

Also can you rebase this so it's mergeable?

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Comment by SkorikSergey** [comment]
ci-test

**Review by rhopp — APPROVED** [review]

**Pull request merged**